src/components/Table/Filters.tsx

```diff
@@ -155,3 +186,14 @@ const ButtonsWrapper = styled.div`
 const ButtonWrapper = styled.div`
   margin-right: 8px;
 `;
+
+interface StyledOutlinedProps {
+  active?: boolean;
+}
+const StyledFilterOutlined = styled(FilterOutlined)<StyledOutlinedProps>`
```

Comment:
I wouldn't be opposed to using `FilterFilled` instead here, but either works!

Reply:
I was unable to find a corresponding SearchFilled from the antd lib so I am going to keep as is.